docs/api/tray.md

```diff
@@ -241,6 +241,15 @@ win.on('hide', () => {
 })
 ```
 
+#### `tray.setIgnoreDoubleClickEvents(ignore)` _macOS_
```

Comment:
@codebytere I think this should be a getter/setter instead. It feels more natural in JS and then you can also check whether it's enabled or not.

```js
tray.ignoreDoubleClickEvents = true;

// Somewhere else in the code-base

if (tray.ignoreDoubleClickEvents) {
```

Reply:
This sounds like an interesting proposal. Is this something that we would want done in this PR? Or could we potentially create a follow up PR which converts one of these instance methods to a getter/setter as a proof of concept?

Reply:
> I realize this goes against many of the other Electron APIs,

Currently the standard is getter and setter methods for All The Things.  If people want to change this then we should raise an issue with a firm proposal on what we want changed.  For  now let's go with setter and getter methods and if we decide in the future to have a new standard on what our apis look like we can come back and address it.

Reply:
> For now let's go with setter and getter methods and if we decide in the future to have a new standard on what our apis look like we can come back and address it.

This PR is missing a getter method.

> and if we decide in the future to have a new standard on what our apis look like we can come back and address it.

https://github.com/electron/electron/issues/12745

Reply:
Thanks for creating that issue! I just added a getter method method to this PR.